content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: Asset Cache Server Mode
+title: Asset Cache Server
+description: Enable a feature to reduce asset builds for a team.
+weight: 200
+toc: true
+---
+
+The **Asset Server Cache** mode is a feature allows an **Asset Processor** to cache product asset files to a remote folder so that other team members can skip long asset processing times.
+
+The **Asset Server Cache** (ACS) mode lets the **Asset Processor** fetch pre-processed products of a job from an asset server cache instead of processing it locally. It does this by using a shared folder that the **Asset Processor** writes out product asset archives ZIP files for a processed source asset job. The other **Asset Processor** clients retrieve these ZIP files and unzip the products assets instead of processing the source asset from scratch.
```

Comment:
```suggestion
The **Asset Server Cache** (ACS) mode lets the **Asset Processor** fetch pre-processed products of a job from an asset server cache instead of processing it locally. It does this by using a shared folder where the **Asset Processor** writes out product asset archive ZIP files for a processed source asset job. The other **Asset Processor** clients retrieve these ZIP files and unzip the product assets instead of processing the source asset from scratch.
```